dbms/src/Core/Settings.h

```diff
@@ -326,7 +326,7 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingBool, external_table_functions_use_nulls, true, "If it is set to true, external table functions will implicitly use Nullable type if needed. Otherwise NULLs will be substituted with default values. Currently supported only for 'mysql' table function.") \
     M(SettingBool, allow_experimental_data_skipping_indices, false, "If it is set to true, data skipping indices can be used in CREATE TABLE/ALTER TABLE queries.") \
     \
-    M(SettingBool, experimental_use_processors, false, "Use processors pipeline.") \
+    M(SettingBool, experimental_use_processors, true, "Use processors pipeline.") \
```

Comment:
hmm, is this intentional and indicating a production ready status of the processor pipeline?

Reply:
oh, that reminds me of toggling the flag on for the sso map too. thanks

Reply:
It is not necessarily if your SSO map is strictly better and well tested.